src/program.js

```diff
@@ -246,6 +246,12 @@ Example: $0 --help run.
       describe: 'Show verbose output',
       type: 'boolean',
     },
+    'ignore-files': {
+      alias: 'i',
+      describe: 'Files to ignore.',
```

Comment:
This should be more explicit and should show an example of ignoring multiple files. It should say that the file must be an absolute path. I think an example of multiple paths would look like this, no? `--ignore-files /path/to/first.js /path/to/second.js`